ckanext/dalrrd_emc_dcpr/templates/snippets/facet_list.html

```diff
@@ -43,36 +42,52 @@ <h2 class="panel-title">
                 <div id="{{ item_id }}" class="panel-collapse collapse" role="tabpanel" aria-labelledby="{{ item_id }}">
                 <div class="panel-body">
 			{% with items = items or h.get_facet_items_dict(name, search_facets or c.search_facets) %}
-			{% if item_id == 'DCPRRequest' %}
-			{% set items = {'Public requests':"", 'My requests':"my-dcpr-requests", 'Requests awaiting NSIF moderation':"awaiting-nsif-moderation-dcpr-requests", 'Requests awaiting CSI moderation':"awaiting-csi-moderation-dcpr-requests", 'Requests under preparation':"under-preparation-dcpr-requests"} %}
-			{% endif %}
+				{% if item_id == 'DCPRRequest' %}
```

Comment:
I think we should use similar logic used in the `dcpr/list.html` template on checking what action links should the current user see.

Reply:
@Samweli can you elaborate more please, after reviewing dcpr/list.html, wasn't sure which checks you're referring to, there is a slight difference how the code approach accessibility, the dcpr/list.html begins with the user and checks it's role (sysadmin, user that has permission to show the relevant section), and accordingly shows navigation tabs, 

the code here begins with the permission itself (dcpr_request_list_public_auth, awaiting-nsif-moderation-dcpr-requests, awaiting-nsif-moderation-dcpr-requests ..etc. ) and checks if the user (sysadmin or a user with permission) has accessibility,

**also the code you showed above is removed** (indicated by the red highlight) and the following is what substituted it (give the PR a double check), 

```
{% set items = {'Public requests':{"link":"", "access":"dcpr_request_list_public_auth"},
'My requests':{"link":"my-dcpr-requests", "access":"my_dcpr_request_list_auth"},
'Requests awaiting NSIF moderation':{"link":"awaiting-nsif-moderation-dcpr-requests", "access":"dcpr_request_list_pending_nsif_auth"},
'Requests awaiting CSI moderation':{"link":"awaiting-csi-moderation-dcpr-requests", "access":"dcpr_request_list_pending_csi_auth"},
'Requests under preparation':{"link":"under-preparation-dcpr-requests", "access":"dcpr_request_list_under_preparation_auth"}}
 %}
```

Reply:
@Mohab25 I meant we should add more conditions and check if user is a NSIF representative and then show the `Requests awaiting NSIF moderation` action link, check if user is a CSI representative and then show the `Requests awaiting CSI moderation` action link, if not the hide the respective links.

Reply:
@Samweli i think i got a sense of what you mean, but should the user be a representative to get access for the requests awaiting NSIF/CSI moderation, or is it sufficient for the user to be a member of NSIF/CSI ?, what i've seen from the current system behavior, you need only to be a member of these organizations to be able to list requests pending by the relative organization, by if you are reviewing a request (i.e. having a reviewer role for a specific request) more permissions might be needed (please tell me if i'm wrong here, i only tested the system for specific functionalities), if this point holds true (you only need to be a member not the actual reviewer to list requests pending by your organization),  look at these checks:
"access":"dcpr_request_list_public_auth"     -- this is public dcpr requests listings (available for registered user), 
"access":"my_dcpr_request_list_auth"          -- these are the requests made by the user (my-requests)
"access":"dcpr_request_list_pending_nsif_auth"  -- this is nsif members
"access":"dcpr_request_list_pending_csi_auth" -- the csi members 
"access":"dcpr_request_list_under_preparation_auth" -- requests under preparation,

now at line 63 i'm checking if the user has these permissions `{% if not h.check_access(items[item]['access']) %}` which results on showing the whole links (like the case of sys admins), or a subset of them.

Reply:
@Samweli any follow-up?, the best way to test it is to use different testers (tester1, tester2, admin ..etc.) with different orgs,  and compare the functionality to the expected one, pretty confident it will match the existing behavior.

Reply:
@Mohab25 looks good to me, now. I was missing to see how you were achieving the functionality but now I see the access functions make it possible to correctly show/hide the required actions link.